floss/main.py

```diff
@@ -56,6 +57,9 @@
 
 logger = floss.logging_.getLogger("floss")
 
+# use rich as default Traceback handler
+install(show_locals=True)
+
```

Comment:
please invoke this from within `main`. otherwise, this configuration is set whenever `floss.main` is imported, which is a problem if when floss is used as a library, since it should be up to the application to decide how to handle output/logging.

Reply:
also, lets use the fully qualified path `rich.traceback.install` so thats its more clear where the name comes from.